gh-badges/lib/color.js

```diff
@@ -0,0 +1,49 @@
+'use strict'
+
+const isCSSColor = require('is-css-color')
+
+// When updating these, be sure also to update the list in `gh-badges/README.md`.
+const namedColors = {
+  brightgreen: '#4c1',
+  green: '#97CA00',
+  yellow: '#dfb317',
+  yellowgreen: '#a4a61d',
+  orange: '#fe7d37',
+  red: '#e05d44',
+  blue: '#007ec6',
+  grey: '#555',
+  gray: '#555',
+  lightgrey: '#9f9f9f',
+  lightgray: '#9f9f9f',
+}
+
+const hexColorRegex = /^([\da-f]{3}){1,2}$/i
```

Comment:
do we need to add an optional `#` to this?

Reply:
after looking through the tests I'm guessing the answer is no since `normalizeColor` handles this?

Reply:
I thought the same thing. However `isCSSColor('#ccc')` is true, so we don't need to accept it here.

Reply:
Ahhh, thanks for that. I was scratching my head on that one